lib/actor.js

```diff
@@ -37,7 +37,10 @@ module.exports = function (obj) {
     });
 
   // add print comment method`
-  obj.say = (msg, color = 'cyan') => recorder.add(`say ${msg}`, () => output.say(msg, `${color}`));
+  obj.say = (msg, color = 'cyan') => recorder.add(`say ${msg}`, () => {
+    event.emit('step.comment', msg);
```

Comment:
```suggestion
    event.emit(event.step.comment, msg);
```
`event.step.comment` should be declared as const inside `events.js`